tests/swagger_generation/registries/hidden_api.py

```diff
@@ -0,0 +1,621 @@
+import marshmallow as m
+
+from flask_rebar import Rebar
+from flask_rebar import HeaderApiKeyAuthenticator
+from flask_rebar import compat
+from flask_rebar.swagger_generation import SwaggerV2Generator, SwaggerV3Generator
+
+rebar = Rebar()
+registry = rebar.create_handler_registry()
+
+swagger_v2_generator = SwaggerV2Generator()
+swagger_v3_generator = SwaggerV3Generator(include_hidden=True)
+hidden_swagger_v3_generator = SwaggerV3Generator()
```

Comment:
Not a showstopper but I found the naming a little confusing and at first thought something was backwards in tests.  Would recommend calling this (the normal) one just `swagger_v3_generator` and the other something like `swagger_v3_generator_with_hidden` to make it explicit and obvious